specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/WebApps.json

```diff
@@ -3243,41 +3268,34 @@
             }
           }
         }
-      },
-      "put": {
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/functions/{functionName}/listsecrets": {
```

Comment:
Examples link?

Reply:
This listsecrets API is not new. However you are right that it doesn't have an examples link, and neither do most of the other App Service APIs. We have an issue tracking that here: https://github.com/Azure/azure-rest-api-specs/issues/6561. We'll address the samples across the board in a separate work item.